### PR TITLE
Tie settings

### PR DESCRIPTION
**Pull request by TordJoranger**
Should make test run again (although without import for now)

### CODE REVIEW

**Comment by TordJoranger** [comment]
IWillfixthetests.com

**Review by jsoi-eq — APPROVED** [review]

File: src/Equinor.ProCoSys.Completion.WebApi/appsettings.json [hunk 1]
```diff
@@ -47,6 +47,14 @@
     "PermissionCacheMinutes": "20",
     "PlantCacheMinutes": "20"
   },
+  "CommonLib": {
+    "CacheDurationDays": 1,
+    "SchemaFrom": [ "TR3111", "Handover", "Work", "ProCoSys_CODETABLE", "Query" ],
```
Comment (janid1967) [inline src/Equinor.ProCoSys.Completion.WebApi/appsettings.json:52]: @jsoi-eq Which scema is punc in? I guess SchemaFrom can be redused to that schema only?

File: src/tests/Equinor.ProCoSys.Completion.WebApi.IntegrationTests/appsettings.json
Comment (janid1967) [inline src/tests/Equinor.ProCoSys.Completion.WebApi.IntegrationTests/appsettings.json:1]: @TordJoranger I think we should try to get rid of this file, insert all its values in the new src/Equinor.ProCoSys.Completion.WebApi/appsettings.IntegrationTests.json and see if that work

Comment (janid1967) [reply src/tests/Equinor.ProCoSys.Completion.WebApi.IntegrationTests/appsettings.json:1]: I can do this later. I Approve

**Review by janid1967 — APPROVED** [review]

File: src/Equinor.ProCoSys.Completion.WebApi/appsettings.json (hunk 1 above)
Comment (janid1967) [reply src/Equinor.ProCoSys.Completion.WebApi/appsettings.json:52]: I talked to J, We found that Punch is in Handover Schema. I push a commit

**Pull request merged**